src/acb_theta/char_dot_acb.c

```diff
@@ -0,0 +1,26 @@
+/*
+    Copyright (C) 2023 Jean Kieffer
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#include "acb_theta.h"
+
+void
+acb_theta_char_dot_acb(acb_t x, ulong a, acb_srcptr z, slong g, slong prec)
+{
+    slong* v;
+
+    v = flint_malloc(g * sizeof(slong));
```

Comment:
Just curious, is it possible to get a non-positive `g` here (based off of the documentation)?